contents/handbook/content-and-docs/docs.md

```diff
@@ -38,28 +38,36 @@ This doesn't preclude anyone from submitting new or improved docs for any produc
 
 | &nbsp                           | **Individual owner** |
 |---------------------------------|----------------------|
-| **Overview**                    | Website & Vibes      |
-| **Start here**                  | Lior                 |
 | **SDKs and frameworks**         | Ian                  |
 | **Reverse proxy**               | Ian                  |
 | **Migrate**                     | Ian                  |
 | **Data**                        | Danilo               |
 | **HogQL**                       | Ian                  |
 | **API**                         | Ian                  |
-| **Alerts**                      | Bijan                |
 
 ### Products
 
 | &nbsp                           | **Individual owner** |
 |---------------------------------|----------------------|
-| **Product analytics**           | Lior                 |
-| **Web analytics**               | Bijan                |
-| **Session replay**              | Bijan                |
+| **Product analytics**           | Ian                  |
+| **Web analytics**               | Andy                 |
+| **Session replay**              | Ian                  |
 | **Feature flags**               | Ian                  |
 | **Experiments**                 | Lior                 |
-| **Surveys**                     | Bijan                |
+| **Surveys**                     | Andy                 |
 | **Data pipelines**              | Danilo               |
-| **Data warehouse**              | Danilo                  |
+| **Data warehouse**              | Danilo               |
+
+### Docs gardener
+
+Each week the Content & Docs crew picks someone to review the state of feedback across these channels:
+
+- #docs-feedback
+- #content-docs-ideas
+- #ask-max for questions missing content
+- Zendesk tickets [where **root cause** is `documentation unclear`](https://posthoghelp.zendesk.com/agent/filters/33465387985947)
+
```

Comment:
Think this would be good to add. Basically the same as #ask-max but for everyone.
```suggestion
- [Inkeep chat sessions](https://portal.inkeep.com/posthog/projects/clz7fyu8i001bomqpr7t8lds8/chat/chat-sessions?filters={%22isOnTopic%22:%22yes%22,%22isClear%22:%22yes%22,%22firstMessageTime%22:%2230d%22,%22isDocumented%22:%22no%22}) where there is a documentation gap

```